codecov/__init__.py

```diff
@@ -982,8 +985,11 @@ def main(*argv, **kwargs):
                 if codecov.gcov_args:
                     cmd.append(sanitize_arg("", codecov.gcov_args or ""))
                 cmd.append(path)
-                write("    Executing gcov (%s)" % cmd)
-                write(try_to_run(cmd))
+                if codecov.gcov_out:
+                    write("    Executing gcov (%s)" % cmd)
```

Comment:
I think we should write this regardless

```suggestion
                write("    Executing gcov (%s)" % cmd)
```

Reply:
A few lines above, it says `Processing gcov..` One of the reasons I thought it might not be necessary is that each of the gcov files is listed under `Collecting reports`.

![image](https://user-images.githubusercontent.com/1364432/101270190-9a7df700-372b-11eb-94cb-9d36507d6c0f.png)

Let me know if you would still like it printed.